src/libLookup/Lookup.cpp

```diff
@@ -130,18 +130,15 @@ bool Lookup::GenTxnToSend(size_t num_txn,
                           uint32_t numShards) {
   vector<Transaction> txns;
 
-  if (GENESIS_KEYS.size() == 0) {
-    LOG_GENERAL(WARNING, "No genesis keys found");
+  if (GENESIS_WALLETS.size() == 0) {
```

Comment:
The question of keys is such that the txn should be signed in gentxn. do we use a spearate constants.xml for that purpose then??

Reply:
Don't we already load the signed transactions from the files?